lib/org/chromium/webidl/Parser.js

```diff
@@ -172,7 +172,7 @@ foam.CLASS({
               InterfaceLike: tseq(
                   alt('interface', 'exception'),
                   sym('identifier'), sym('Inheritance'), '{',
-                  sym('InterfaceMembers'), '}', ';'),
+                  sym('InterfaceMembers'), '}', sym('SemiColon')),
```

Comment:
Rules had to be relaxed for Interface declarations as many found in WebKit IDL files do not end with a semicolon